apps/admin-x-settings/src/components/settings/membership/tiers/TierDetailModal.tsx

```diff
@@ -268,7 +268,7 @@ const TierDetailModalContent: React.FC<{tier?: Tier}> = ({tier}) => {
                                 <TextField
                                     disabled={!hasFreeTrial}
                                     hint={<div className='mt-1'>
-                                    Members will be subscribed at full price once the trial ends. <a className='text-green' href="https://ghost.org/" rel="noreferrer" target="_blank">Learn more</a>
+                                    Members will be subscribed at full price once the trial ends. <a className='text-green' href=" https://ghost.org/help/free-trials/" rel="noreferrer" target="_blank">Learn more</a>
```

Comment:
looks like there's an accidental space in front of the href, but otherwise looks good to me! :)